src/applications/ivc-champva/10-10D/tests/e2e/fixtures/data/app-sa-cs-medab.json

```diff
@@ -0,0 +1,92 @@
+{
+  "description": "Is applicant, sponsor alive, child (step), has Medicare A/B",
```

Comment:
This property summarizes the key points of this data-set. It is also reflected in the filename (abbreviated)

Reply:
This is done for the other datasets as well.